dandi/girder.py

```diff
@@ -157,7 +156,7 @@ def dandi_authenticate(self):
                     "Account/API keys "
                     "in Girder) for {}: ".format(client_name)
                 )
-                keyring.set_password(app_id, "key", api_key)
+                keyring_backend.set_password(app_id, "key", api_key)
```

Comment:
could you please mockup a unittest for this logic? I know that it is old code, but I forgot that we did not test it (due to env var making it return above I guess).

Reply:
Done.